src/Pages/Dashboard.js

```diff
@@ -48,14 +52,14 @@ function DashboardPage(props) {
 }
 
 function loadingStock() {
-  return function WithLoadingComponent({ currentStock, setCurrentStock, ...props}) {
+  return function WithLoadingComponent({ currentStock, setCurrentStock, loggedUsername, loggedIn, ...props}) {
```

Comment:
Same here - when you pass to us we don't need loggedIn for the back end

Reply:
will do!